src/actions/callLogIn.js

```diff
@@ -25,6 +26,8 @@ export const callLogIn = (e, userInput) => {
         });
         console.log("RESPONSE: ", res);
         localStorage.setItem("jwt", res.data.token);
+        history.push('/conversations');
+        window.location.reload();
```

Comment:
is this needed? normally with history.push, we wouldn't need to reload the window, it should do that automatically.